File: exam/task03/src/main/python/task03.py
from pyspark import SparkContext,SparkConf
from pyspark.sql import HiveContext

conf = SparkConf().setAppName("Solution3")
sc = SparkContext(conf=conf)


# TO-DO : READ DATA FROM HDFS
data = sc.textFile("hdfs:///user/hive/warehouse/employees")       

#TO-DO : CHANGE DATA FORMAT

changed = data.map(lambda row : row.replace(',','\t'))

#to-do : save data

changed.saveAsTextFile("hdfs:///user/cloudera/solution3")

sc.stop()

File: exam/task03/src/main/scala/task03.scala
import org.apache.spark.SparkContext
import org.apache.spark.SparkContext._
import org.apache.spark.SparkConf

object task03 {

	def main(args:Array[String]){
	
	val sc = new SparkContext(new SparkConf().setAppName("task03"))
	
	val file = sc.textFile("hdfs:///user/hive/warehouse/employees.db/employees")
	
	val filemap = file.map( row => row.replaceAll(",","\t"))

	filemap.saveAsTextFile("solution3")

	sc.stop()
		

	}
}
